src/org/usfirst/frc4904/robot/commands/SetOverride.java

```diff
@@ -0,0 +1,25 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class SetOverride extends Command {
+	public boolean override;
+	public boolean setValue;
+	public OverridableCommand command;
```

Comment:
You can make both `setValue` and `command` final, and also they should probably be protected.